graylog2-server/src/test/java/org/graylog2/indexer/indices/IndicesTest.java

```diff
@@ -215,7 +215,7 @@ public void testCreateOverwritesIndexTemplate() throws Exception {
             "_source", ImmutableMap.of("enabled", false),
             "properties", ImmutableMap.of("message",
                 ImmutableMap.of(
-                    "type", "binary",
+                    "type", "string",
```

Comment:
It looks like this is related to elastic/elasticsearch#17567 and elastic/elasticsearch#17568.

The test will fail with the following exception if the field type changed:
```
MapperParsingException[Failed to parse mapping [message]: mapper [message] has different [index] values from other types of the same index
]; nested: IllegalArgumentException[mapper [message] has different [index] values from other types of the same index];
```